js/src/lib/components/InferenceWidget/widgets/ImageClassificationWidget/ImageClassificationWidget.svelte

```diff
@@ -106,6 +112,17 @@
 		output = [];
 		outputJson = "";
 	}
+
+	onMount(() => {
+		async () => {
+			const [src] = getDemoInputs(model, ["src"]);
+			if (callApiOnMount && src) {
+				imgSrc = src;
+				const blob = await getBlobFromUrl(imgSrc);
+				getOutput(blob);
+			}
+		};
```

Comment:
I think it needs to be called no? Here it's just defined?

Reply:
oh indeed, handled in 54cf71564356fd98a7fad36121d13029db276b29
thanks for catching it!